src/editor/editor.ml

```diff
@@ -217,6 +217,18 @@ let () =
   end;
 
   (*-------question pane  -------------------------------------------------*)
+
+  let override_url = function
+    | Omd_representation.Url(href,[Omd_representation.Text(text)],title) ->
```

Comment:
three small comments:

* you may want to do `let open Omd_representation in` to avoid writing several times `Omd_representation.`
* I believe the code won't work in more complex situations, where the link contains (nested) Markdown code:
  `[some **bold** or `teletype text`](http://example.com)` so could you try doing some kind of "recursive call" by copying more code from Omd here?
* Finally we might want to add `rel="noopener noreferrer"` as well, in addition to `target="_blank"`, see https://developer.mozilla.org/fr/docs/Web/HTML/Element/a#s%C3%A9curit%C3%A9_et_vie_priv%C3%A9e